src/app.hooks.js

```diff
@@ -125,14 +125,15 @@ const displayInternRequests = (level) => (context) => {
  */
 const errorHandler = (context) => {
 	if (context.error) {
-		// too much for logging...
 		if (context.error.hook) {
+			// too much for logging...
 			delete context.error.hook;
 		}
 
-		// statusCode is return by extern services / or mocks that use express res.status(myCodeNumber)
-		if (!context.error.code && !context.error.statusCode) {
-			context.error = new GeneralError(context.error.message || 'server error', context.error.stack || '');
+		if (!context.error.type || context.error.type !== 'FeathersError') {
```

Comment:
```suggestion
		if (!context.error.type || typeof context.error.type !== 'FeathersError') {
```
is there a typecheck result not working?

Reply:
No it should match if String is set, or not.